tests/callbacks/test_callback_hparams.py

```diff
@@ -12,10 +12,11 @@
 from composer.loggers.logger_hparams_registry import ObjectStoreLoggerHparams, logger_registry
 from composer.profiler import JSONTraceHandler, SystemProfiler, TorchProfiler, TraceHandler
 from tests.callbacks.callback_settings import get_cb_hparams_and_marks, get_cb_kwargs, get_cbs_and_marks
-from tests.common.hparams import assert_in_registry, construct_from_yaml
+from tests.hparams.common import assert_in_registry, construct_from_yaml
```

Comment:
[nit] Is there a bit more consolidation of hparams tests possible here?  We could move this test to the `tests/hparams` area so that everything that calls `tests.hparams.common` lives in one place?

Reply:
ah good point let me move as well.